cloudflare.go

```diff
@@ -171,6 +179,16 @@ query ($zoneIDs: [String!], $time: Time!, $limit: Int!) {
 					datetime
 				}
 			}
+			firewallEventsAdaptiveGroups(limit: $limit, filter: {datetime_geq: $mintime, datetime_lt: $maxtime}) {
```

Comment:
Note on this: IMO it is necessary here to use a time range, instead of just a filter like in `httpRequests1mGroups`, as these metrics are not rolled up minutely by Cloudflare for us. Using only a single timestamp would mean getting only the slice of data with matching timestamps, which is not the whole minute.
In extend, I think the same should be applied to `httpRequestsAdaptiveGroups`, but I am not entirely sure. Happy to hear your thoughts.

Reply:
Hey, I think that's a good way to approach this. Let's add it to `httpRequestsAdaptiveGroups` as well.